includes/class-wp-job-manager-usage-tracking-data.php

```diff
@@ -62,14 +62,23 @@ public static function get_usage_data() {
 			'jobs_by_guests'              => self::get_jobs_by_guests(),
 		];
 
-		$all_extenstions     = self::get_official_extensions( false );
+		$settings = self::get_settings_data();
+
+		foreach ( $settings as $name => $value ) {
+			$name                              = preg_replace( '/[^a-z0-9]/', '_', $name );
+			$usage_data[ 'settings_' . $name ] = preg_replace( '/[^a-z0-9]/', '_', $value );
+		}
+
+		$all_extensions      = self::get_official_extensions( false );
 		$licensed_extensions = self::get_official_extensions( true );
 
-		$usage_data['official_extensions'] = count( $all_extenstions );
+		$usage_data['official_extensions'] = count( $all_extensions );
 		$usage_data['licensed_extensions'] = count( $licensed_extensions );
 
-		foreach ( array_keys( $all_extenstions ) as $installed_plugin ) {
-			$usage_data[ $installed_plugin ] = isset( $licensed_extensions[ $installed_plugin ] ) ? 'licensed' : 'unlicensed';
+		foreach ( array_keys( $all_extensions ) as $installed_plugin ) {
+			$name                = preg_replace( '/[^a-z0-9]/', '_', $installed_plugin );
```

Comment:
This will make it practically impossible to track historical data for this stat I think

Reply:
Sorry didn't explain this well, this one is a fix: Tracks is rejecting the events with these plugin fields because the keys are invalid, they have to be snake_case.

Reply:
So this never worked? Oh my! I remember testing it but I didn't do it end to end it seems..